Microphone.Nancy/NancyProvider.cs

```diff
@@ -6,14 +6,13 @@
 
 namespace Microphone.Nancy
 {
-    public class NancyProvider : IFrameworkProvider
+    public class NancyProvider<TBootstrapper> : IFrameworkProvider where TBootstrapper : DefaultNancyBootstrapper
```

Comment:
I think `: DefaultNancyBootstrapper` should be `: INancyBootstrapper` instead as that is what is assigned in the `new NancyHost(...`